DataRepo/tests/test_infusate_name_parser.py

```diff
@@ -107,3 +108,57 @@ def test_malformed_infusate_parsing_2(self):
         name = "not a properly encoded tracer name"
         with self.assertRaisesRegex(TracerParsingError, "cannot be parsed"):
             _ = parse_infusate_name(name)
+
+    def test_malformed_infusate_parsing_3(self):
+        # Test back-to-back occurrences of curlies expressions
+        name = "short_name{tracers1}{tracers2}"
```

Comment:
The examples in the notes were not literal strings.  The test entries in the issue owner section are the literals.  I only supplied the notes for background.  So I'll create suggestions to enter the strings I came uo with in the issue owner tests section...
```suggestion
        name = "myshortname{lysine-[13C5]}{glucose-[13C4]}"
```